account_move_name_sequence/models/account_move.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# @author: Moisés López <moylop260@vauxoo.com>
+# @author: Francisco Luna <fluna@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    name = fields.Char(compute="_compute_name_by_sequence")
```

Comment:
Not stored anymore?

CC @moylop260

Reply:
@luisg123v 

It is a field already defined in the original model

Here only changing the custom changes in this case the compute parameter

string, store and more parameters are the original ones from the super class